### ISSUE TITLE
NzbFox and Transmission Easy Client conflicts with VimFx on pinned or restored tabs

### ISSUE THREAD

**Issue by Aldarone** [akhodakivskiy/VimFx#693]
# Steps to reproduce:

1. Create a new Firefox Profile
2. Install VimFx
3. Install one of [NzbFox][2] or [Transmission Easy Client][3]
4. Restart Firefox
5. Login to Facebook or Tweetdeck
6. Click on an input field

# Description:

VimFx should enter in Insert Mode (with the grey icon) and I should be able to type any text. Instead, it stays in Normal Mode and the shortcuts are activated when I type.

This bug only happens when e10s is enabled (by default in Developer Edition 46).

VimFx version: 0.12.0
Firefox version: Developer Edition, 46.0a2 (2016-01-26) - e10s enabled
OS: Archlinux x64
Keyboard language: Fr
Worked before: Yes (It worked with Firefox Beta 43)

# Example URL(s):
- https://www.facebook.com/
- https://tweetdeck.twitter.com/
- https://www.toggl.com/app/timer

[2]: https://addons.mozilla.org/en-US/firefox/addon/nzbfox/
[3]: https://addons.mozilla.org/en-US/firefox/addon/transmission-easy-client/

**Comment by lydell** [comment]
Thanks for your excellent bug report!

Unfortunately, I cannot reproduce your problem.

VimFx version: 0.12.0, latest master, latest develop
Firefox version: 47.0a1 (2016-01-26) – e10s enabled. Weird that we’ve got the same date but different version numbers.
OS: Trisquel 7
Keyboard language: en-US

**Comment by Aldarone** [comment]
Thank you for trying !

I searched for a simpler case to reproduce and I think I found one. It seems to happen only on pinned / restored tabs or the default home page.

1. Create a new Firefox Profile
2. Install VimFx
3. Install one of [NzbFox][2] or [Transmission Easy Client][3]
4. Pin a tab (like google.com or whatever)
5. Restart Firefox
6. Click on an input field

As soon as you close the buggy tab and un-close it, it works again.

I made a screencap to show you : https://youtu.be/YzpxGseOqPE
